Annotato/Annotato/ViewController.swift

```diff
@@ -0,0 +1,181 @@
+//
```

Comment:
What is this `ViewController` supposed to be? Is this supposed to be a `PDFImportViewController`? Can you put it in a named folder under views and rename this because it's a little confusing. I also can't find the storyboard attached to it if there's supposed to be one since there are `IBOutlet`